user_docs/en/userGuide.t2t

```diff
@@ -2589,33 +2589,67 @@ To toggle the braille viewer from anywhere, please assign a custom gesture using
+When exiting the Add-on Store, you may need to restart NVDA so that your changes can take effect.
+NVDA will ask you to restart when exiting the Add-on Store if you have updated, installed, removed or changed the status of an add-on.
 
-To browse and download available add-ons online, press the Get add-ons button.
-This button opens the [NVDA Add-ons page https://addons.nvda-project.org/].
-If NVDA is installed and running on your system, you can open the add-on directly from the browser to begin the installation process as described below.
-Otherwise, save the add-on package and follow the instructions below.
+To access the Add-on Store from anywhere, please assign a custom gesture using the [Input Gestures dialog #InputGestures].
 
-To install an Add-on you previously obtained, press the Install button.
-This will allow you to browse for an add-on package (.nvda-addon file) somewhere on your computer or on a network.
++++ Browsing add-ons +++[AddonStoreBrowsing]
+The Add-on Store shows a list of add-ons in a panel.
+By default, the list will be currently installed add-ons.
+To browse and download available add-ons online, change the "Filter by Status" (``alt+f``) to "Available add-ons".
+
+Selecting an add-on will display the details for the add-on in the next panel.
+Associated actions will also become available for the add-on: e.g. to install, update, disable, enable or remove the add-on.
+If there is help available for an installed add-on, you can access it by pressing the "Add-on help" button (``alt+h``).
+
+++++ Filter add-ons by status ++++[AddonStoreFilterStatus]
+To change the view of add-ons, change the "Filter by Status" (``alt+f``).
+You can filter by installed, updatable, disabled or available add-ons.
+To include outdated [incompatible add-ons #incompatibleAddonsManager] in available add-ons, enable the [relevant setting #IncompatibleAddonsSetting] in NVDA preferences.
+
+++++ Filter add-ons by channel ++++[AddonStoreFilterChannel]
+To change the selection of add-ons, change the "Channel" filter selection (``alt+n``).
+
+There are 3 channels of add-ons:
+- Stable: The developer has released this as a tested add-on with a released version of NVDA.
+- Beta: This add-on may need further testing.
+Suggested for early adopters.
+- Dev: This channel is suggested to be used by add-on developers to test unreleased API changes.
+NVDA alpha and beta testers may need to use a "Dev" version of their add-ons.
+-
+
+++++ Search add-ons ++++[AddonStoreFilterSearch]
+Add-ons can be filtered by display name, publisher and description.
+To search add-ons, use the "Search" text box (``alt+s``).
+
++++ Installing add-ons +++[AddonStoreInstalling]
+It is very important to only install add-ons from sources you trust.
+The functionality of add-ons is unrestricted inside NVDA. 
+This could include accessing your personal data or even the entire system.
+
+You can install and update add-ons by [browsing the Add-on Store #AddonStoreBrowsing].

```

Comment:
Perhaps a slightly longer description here talking about setting the filter to "Installed", select an add-on in the list and tab to the Install button.

Reply:
Might be worth reiterating the sentence from the start of the Browse section: "To browse and download available add-ons online, change the "Filter by Status" (alt+f) to "Available add-ons".", then saying "Once an add-on has been selected, the right side of the screen will display a description.  Press tab to move to and read the description.  To install the add-on, activate the "Install" button under the description, or press alt+i.